linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/ansible/shared.yml

```diff
@@ -4,19 +4,49 @@
 # complexity = low
```

Comment:
Given https://github.com/ComplianceAsCode/content/commit/e7d99c03246ebc5eacba43156987e3548a6ea73e

Please ensure that `multi_platform_ubuntu` is added

Reply:
I think you could consolidate also the bash remediation into a single shared file

Reply:
thank you